squiggy/lib/util.py

```diff
@@ -31,6 +31,8 @@ def isoformat(value):
 
 
 def to_int(s):
+    if not s:
+        return None
```

Comment:
@pkerschen - If `s` is zero we want to return zero, not None?

Reply:
@johncrossman Oh, right, thanks. I was trying to catch None input (which throws a TypeError), but that's better handled by catching TypeError below as well as ValueError. I'll amend.